excelize.go

```diff
@@ -25,7 +25,8 @@ import (
 	"strconv"
 	"strings"
 	"sync"
-
+	"net/http" 			// https://github.com/ahah43
+"net/url"				// https://github.com/ahah43
```

Comment:
The code style should be following the [Contributing Guidelines](https://github.com/xuri/excelize/blob/master/CONTRIBUTING.md), it should be formatted, and remove the comments.